INSTALL.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# 
+# 
+# Installation script for PyKaldi
+# 
+# Usage:
+# 	./INSTALL.sh [python] [pip] [python_library]
+# 
+# 	python, pip (optional) point to the location of the executables. Defaults to $(which ..)
+# 	python_library (optional) location of the libpython shared library. As default runs a script to find it.
+# 
+# 
+
+set -e -x 
+INSTALL_NINJA=true
+
+#######################################################################################################
+# Installation configuration
+# This determine where things are going to get installed
+#######################################################################################################
+PYKALDI_DIR="$PWD"
+TOOLS_DIR="$PYKALDI_DIR/tools/extras"
+PROTOBUF_DIR="$TOOLS_DIR/protobuf"
+# NINJA_DIR="$TOOLS_DIR/ninja"
+CLIFSRC_DIR="$TOOLS_DIR/clif"
+export KALDI_DIR="$TOOLS_DIR/kaldi"
```

Comment:
no need to export this if kaldi is installed in tools/kaldi. setup.py now checks if kaldi is there.